lib/movements.js

```diff
@@ -1,94 +1,92 @@
-
-const Vec3 = require('vec3').Vec3
+const { Vec3 } = require('vec3')
 const nbt = require('prismarine-nbt')
-const Move = require('./move')
-
-const cardinalDirections = [
-  { x: -1, z: 0 }, // north
-  { x: 1, z: 0 }, // south
-  { x: 0, z: -1 }, // east
-  { x: 0, z: 1 } // west
-]
-const diagonalDirections = [
-  { x: -1, z: -1 },
-  { x: -1, z: 1 },
-  { x: 1, z: -1 },
-  { x: 1, z: 1 }
-]
+const _minecraftData = require('minecraft-data')
+const { cardinalDirections, diagonalDirections } = require('./constants')
+const { Move } = require('./Move')
 
 class Movements {
-  constructor (bot, mcData) {
-    this.bot = bot
+  constructor (bot, minecraftData = _minecraftData(bot.version)) {
```

Comment:
It is better to encourage people to give mcData rather than creating a new one behind their back. There are plans to have a single instance of mcData available in the bot, so this could be simplified, but not this way.